### PR TITLE
Update "stability" scripts to use Google Chrome instead of Chromium

### PR DESCRIPTION
**Pull request by jugglinmike**
Note that this changeset includes a meaningless test modification for
demonstration purposes. That should probably be removed prior to merging
(although leaving it as-is should be harmless).

### CODE REVIEW

File: ci_stability.sh [hunk 1]
```diff
@@ -20,13 +20,24 @@ hosts_fixup() {
     echo "travis_fold:end:hosts_fixup"
 }
 
+install_chrome() {
+    channel=$1
+    deb_archive=google-chrome-${channel}_current_amd64.deb
+    wget https://dl.google.com/linux/direct/$deb_archive
+    sudo dpkg --install $deb_archive || true
+    sudo apt-get install --fix-broken
```
Comment (jgraham) [inline ci_stability.sh]: It's not quite clear why this is needed. Leave a comment, perhaps?

File: url/url-origin.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (jgraham) [inline url/url-origin.html]: Remove this change.

File: ci_stability.sh [hunk 3]
```diff
@@ -20,13 +20,24 @@ hosts_fixup() {
...
+}
 
 test_stability() {
     python check_stability.py $PRODUCT
 }
 
 main() {
     hosts_fixup
+    if [ $(echo $PRODUCT | grep '^chrome:') ]; then
+       install_chrome $(echo $PRODUCT | grep --only-matching '\w\+$')
```
Comment (jgraham) [inline ci_stability.sh]: I guess I might have used two variables rather thing string manipulation, but this is OK for now.

File: ci_stability.sh (hunk 1 above)
Comment (jugglinmike) [inline ci_stability.sh]: Sure. I've also refactored this with an `if` statement to make the relationship between the commands more clear.

File: url/url-origin.html (hunk 2 above)
Comment (jugglinmike) [inline url/url-origin.html]: Done

**Comment by jgraham** [comment]
Mind gunning `git rebase --autosquash origin/master` before landing?

**Review by jgraham — APPROVED** [review]

**Comment by jugglinmike** [comment]
Sure thing. The prior version of this branch is available at [jugglinmike/chrome-presquash](https://github.com/w3c/web-platform-tests/compare/73f4ad361ca56d890b5f46d62c3bc84371a77e10...jugglinmike:chrome-presquash) for posterity.

**Pull request merged**